autoshutdown/core.py

```diff
@@ -142,6 +142,12 @@ def power_action(self):
             self.os_hibernate()
         elif self.config["system_state"] == 'suspend':
             self.os_suspend()
+        elif self.config["system_state"] == 'close':
+            self.close_app()
+
+    def close_app(self):
+        log.info("[AutoShutDown] Closing Deluge...")
+        component.get('MainWindow').quit(shutdown=True)
```

Comment:
If anyone could add some dialog with a countdown and "cancel" button, it would be nice.